network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -438,6 +439,58 @@ class ParticipateHighlights2(ParticipateHighlightsBase):
     )
 
 
+@register_snippet
+class FocusArea(models.Model):
+    interest_icon = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name='interest_icon'
+    )
+
+    name = models.CharField(
+        max_length=100,
+        help_text='The name of this area of focus. Max. 100 characters.',
+    )
+
+    description = models.TextField(
+        max_length=300,
+        help_text='Description of this area of focus. Max. 300 characters.',
+    )
+
+    url = models.URLField(
+        max_length=2048,
+        help_text='URL for this area of focus\' dedicated page.',
+    )
+
+    panels = [
+        ImageChooserPanel('interest_icon'),
+        FieldPanel('name'),
+        FieldPanel('description'),
+        FieldPanel('url'),
+    ]
+
+    def __str__(self):
+        return self.name
+
+    class Meta:
+        verbose_name = 'Area of focus'
+        verbose_name_plural = 'Areas of focus'
+
+
+class AreaOfFocus(WagtailOrderable, models.Model):
```

Comment:
I don't believe we need `models.Model` in here as `Orderable` is a `Model` already.

Reply:
yep, good catch!

Reply:
```suggestion
class AreaOfFocus(WagtailOrderable):
```